reposcan/reposcan.py

```diff
@@ -256,11 +256,16 @@ def parse_repolist_json(cls, data):
         seen = set()
         for repo_group in data:
             # Entitlement cert is optional, use default if not specified in input JSON
-            if "entitlement_cert" in repo_group:
+            if "entitlement_cert" in repo_group and isinstance(repo_group['entitlement_cert'], object):
```

Comment:
Are you sure?

```
>>> isinstance({}, object)
True
>>> isinstance(False, object)
True
```

shouldn't it be `dict`?
```
>>> isinstance({}, dict)
True
>>> isinstance(False, dict)
False
```

Reply:
You're right, fixed.